oat-sa/extension-tao-itemqti
# Hotfix/bsa 370/rollback inline img

related to https://oat-sa.atlassian.net/browse/BSA-370

Description
Rollback of changes introduced for May release by https://oat-sa.atlassian.net/browse/AUT-2673

How to test
Check description of the ticket and check display of item is back to desired by customer
Kitchen env is referred in the Environment field of the ticket

⚠️ update to release version of `tao-item-runner-qti-fe`
